app/dto/transaction.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel
```

Comment:
I've separated Data transfer layer from models layer, so further changes whether needed for the api consumer side or database side are easier to execute.